overlay/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+nav_order: 8
+title: Overlays
+has_children: true
+has_toc: false
+---
+
+# Introduction to OpenAPI Overlay Specification
+
+The [Overlay Specification](https://spec.openapis.org/overlay/latest.html) defines a document format for information that transforms an existing OpenAPI description yet remains separate from the OpenAPI description's source document(s).
+The goal of the Overlay Specification is to provide a mechanism for providing consistent, deterministic updates to a given OpenAPI description, as an aid to automation throughout the API lifecycle.
```

Comment:
```suggestion
The Overlay Specification defines a mechanism for providing consistent, deterministic updates to a given OpenAPI description, as an aid to automation throughout the API lifecycle.
```
To avoid double "provide".